src/Builder/ArchiveBuilderHelper.php

```diff
@@ -87,6 +99,12 @@ protected function isOneOfNamesInList(array $names, array $list): bool
         return false;
     }
 
+    /**
+     *
+     * @param string $name
```

Comment:
Some `@param` and `@return` annotations are redundant with method signature; is it really required by phpstan?

Reply:
I don't believe so.

These was a [discussion about the superfluous phpdoc rule](https://github.com/composer/satis/discussions/679); but so far consensus was to keep the rule in place.

Reply:
It is also more consistent for the time being.
And to be honest:
Removing some and adding others is usually counterproductive in readability, it really confuses people as to the positioning of those lines then if they ever need to be added.